### ISSUE TITLE
source 

### ISSUE THREAD

**Issue by Lucifer0601code** [Killerpac/Kayuri#125]
i knew a developer and he is into making source for an app so i guess you guys can contribute? do you have a disc?

**Comment by Sethnakulit** [comment]
Yes this is much better i think we can provide premium account. Kayuri is much better than other apps
